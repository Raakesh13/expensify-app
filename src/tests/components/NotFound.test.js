import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import toJSON from "enzyme-to-json";

import NotFoundPage from "../../components/NotFound";

configure({ adapter: new Adapter() });

test("should render the nonfound component", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
