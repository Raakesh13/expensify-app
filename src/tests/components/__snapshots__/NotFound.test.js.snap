// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the nonfound component 1`] = `
<div>
  404!
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
